modules/querier/http.go

```diff
@@ -22,6 +23,8 @@ const (
 	BlockStartKey = "blockStart"
 	BlockEndKey   = "blockEnd"
 	QueryModeKey  = "mode"
+	TimeStartKey  = "timeStart"
```

Comment:
i'd prefer these params be called `start` and `end`. that would match the same parameters on the `/api/search` endpoint.